tests/unit_tests/data_steward/validation/participants/identity_match_test.py

```diff
@@ -369,8 +370,8 @@ def test_match_participants_same_participant_simulate_write_errors(self):
     def test_match_participants_same_participant_simulate_location_pii_read_errors(
         self):
         # pre conditions
-        self.mock_location_pii.side_effect = googleapiclient.errors.HttpError(
-            500, b'bar', b'baz')
+        self.mock_location_pii.side_effect = test_util.mock_google_http_error(
+            status_code=500, content=b'baz', reason='bar')
```

Comment:
what does this `test_util` object give us that is better than just using python mock objects?  This same comment applies to the other areas replacing mocks that are set to actual objects with this faked test_util object.

Reply:
I'm not sure I understand the question.  The `HttpError` object now must be given a `requests.Response` instance at construction, preventing all existing definitions from working.  To that end, this func builds an upstream `HttpError` instance with a valid `requests.Response` instance that can be used with the mock funcs, as is demonstrated here.

Perhaps we should have a discussion as to what you would propose as an alternative?